plugins/woocommerce-admin/client/payments-welcome/index.tsx

```diff
@@ -68,12 +88,19 @@ const ConnectAccountPage = () => {
 		}
 	};
 
-	const handleSetup = async () => {
-		setSubmitted( true );
+	const trackConnectAccountClicked = () => {
 		recordEvent( 'wcpay_connect_account_clicked', {
 			wpcom_connection: isJetpackConnected ? 'Yes' : 'No',
 			incentive_id: incentive.id,
+			path: 'payments_connect_core_test',
```

Comment:
I see, you reused this https://github.com/woocommerce/woocommerce/pull/48901/files#diff-a5e6812c40fd62fa1b7df3a3d63dfcb6921d2ae439a8dc7d2f98306da58bef99R62

Reply:
This is how the incentives page was identified historically (probably because in 2022, the incentives started as an experiment). To maintain our data cohesion, we kept it the same when we revamped our incentives in 2023.